test/integration/transactions-convenient-api/transactions-convenient-api.spec.test.ts

```diff
@@ -0,0 +1,28 @@
+import * as path from 'path';
+
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+const SKIPPED_TESTS = [
+  'callback succeeds after multiple connection errors',
+  'callback is not retried after non-transient error',
+  'callback is not retried after non-transient error (DuplicateKeyError)'
+];
+
+describe('Transactions Convenient API Spec Unified Tests', function () {
+  this.beforeEach(function () {
+    if (this.configuration.topologyType === 'LoadBalanced') {
+      if (this.currentTest) {
+        this.currentTest.skipReason =
+          'TODO(NODE-5931) - Fix socket leaks in load balancer transaction tests.';
+      }
+    }
+    this.skip();
+  });
+
+  runUnifiedSuite(loadSpecTests(path.join('transactions-convenient-api', 'unified')), test => {
+    return SKIPPED_TESTS.includes(test.description)
+      ? 'TODO(NODE-): Skipping failing transaction tests'
```

Comment:
Is this for the tests failing related to https://jira.mongodb.org/browse/DRIVERS-2816? Maybe we can skip or retry based on the outcome of the test? I know neither is easy given the structure of the unified runner... The Mocha context is passed to the runner and would be the same context given to an afterEach hook, maybe there's a way to communicate "unfailing" the test that way?

Reply:
@nbbeeken Hold off for review again I still see I have a few more to skip individually.